src/TitleDetails/PiecesList/PiecesList.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { MultiColumnList } from '@folio/stripes/components';
+import { FolioFormattedDate } from '@folio/stripes-acq-components';
+
+const columnMapping = {
+  title: <FormattedMessage id="ui-receiving.piece.title" />,
+  caption: <FormattedMessage id="ui-receiving.piece.caption" />,
+  format: <FormattedMessage id="ui-receiving.piece.format" />,
+  receiptDate: <FormattedMessage id="ui-receiving.piece.receiptDate" />,
+  receivedDate: <FormattedMessage id="ui-receiving.piece.receivedDate" />,
+};
+
+const PiecesList = ({ pieces, title, visibleColumns }) => {
+  const formatter = {
+    title: () => title,
```

Comment:
If I remember correctly, you could just omit such things

Reply:
Unfortunately no, title it's a prop but not a piece property